packages/pharos/src/components/button/pharos-button.ts

```diff
@@ -271,7 +271,7 @@ export class PharosButton extends ScopedRegistryMixin(FocusMixin(AnchorElement))
       icon = this.icon;
     }
 
-    return icon ? html` <pharos-icon name="${icon}"></pharos-icon> ` : nothing;
+    return icon ? html` <pharos-icon name="${icon}" a11y-hidden="true"></pharos-icon> ` : nothing;
```

Comment:
For context, this isn't an actual change in the rendered component, as previously any icon with no description would be set to aria-hidden="true" when rendered. This just codifies it in a more obvious way. I would propose if we want to change that behavior that be a followup issue/pr. I also think @sirrah-tam might have revised this logic in the v14 branch

Reply:
Currently for icon-only buttons we don't have the ability to add alternative text to the icon itself and rather have had to rely on using the `label` (now `a11y-label`) attribute to give the button it's accessible name. We didn't change anything around this for v14 but I think it still works pretty well. The icon-only button setup is such that consumers only have to indicate the icon name on the button component for it to appear, so they aren't thinking about the icon component at all. They'd then just need to remember to ensure that the button has some way of calculating an accessible name for the button which _hopefully_ would be flagged by an automated accessibility test if they don't add the `a11y-label`. We could add some error logic around this to prevent that from happening, but maybe that is part of an overall accessibility awareness training.

Reply:
Cool, all sounds good to me! Thanks for opining.